src/types/identity.ts

```diff
@@ -0,0 +1,50 @@
+import { body, ValidationChain } from "express-validator";
+import { nameof } from "./util/nameof";
+import { VALID_LOC_VALUES } from "../constants";
+
+export default interface Identity {
+  vot?: string;
+  vtm?: string;
+  iss?: string;
+  sub?: string;
+  nbf?: number;
+  exp?: number;
+  aud?: string;
+  iat?: number;
+  jti?: string;
```

Comment:
Feels like we should be setting some sensible defaults for some of these?

Reply:
As above: vot and vc are configurable. The other fields are now generated.